src/ngResource/resource.js

```diff
@@ -363,6 +427,22 @@ angular.module('ngResource', ['ng']).
       }
     };
 
+    /**
+     * @ngdoc method
+     * @name $resourceProvider#$get
+     * @description Method used by angular DI system to create a `$resourceProvider` instance.
+     *
+     * User applications shouldn't need to call this method directly.
+     * This shows the fact that {@link ngResource.$resource $resource}
+     * is a high level service built by leveraging {@link ng.$http $http} service.
+     *
+     * @param {Object} $http An instance of {@link ng.$http $http} service.
+     *
+     * @param {Object} $q An instance {@link ng.$q $q}, Angular's implementation of deferred API.
+     *
+     *
+     * @returns {Function} A factory function, when invoked, returns a $resource object.
+     */
```

Comment:
`$get` methods should not be documented. They are idiomatic and kind of private to Angular.
As you mention, they should only be called by the framework itself.